src/Sentry/Internal/Hub.cs

```diff
@@ -125,6 +131,17 @@ public void WithScope(Action<Scope> scopeCallback)
                 transaction.Sdk.AddPackage(protocolPackageName, nameAndVersion.Version);
             }
 
+            // Release information
+            transaction.Release ??= _options.Release ?? ReleaseLocator.GetCurrent();
+
+            // Environment information
+            var foundEnvironment = EnvironmentLocator.Locate();
```

Comment:
Good for us to GA this on time but good reminder that we need to get back to this after GA.